js/ui/tool/select.js

```diff
@@ -101,6 +101,10 @@ const selectTransformTool = () =>
 
 				// Selection bounding box object. Has some witchery to deal with handles.
 				const selectionBB = (x1, y1, x2, y2) => {
+					x1 = Math.round(x1);
+					y1 = Math.round(y1);
+					x2 = Math.round(x2);
+					y2 = Math.round(y2);
 					return {
```

Comment:
lol also don't think i ever noticed any issue here either, i almost exclusively leave grid snapping turned on

Reply:
Yeah, it was having some problems when selecting things with no grid snapping:

https://user-images.githubusercontent.com/7552751/206338691-4091a3e8-9718-498c-828e-107d3a7a006d.mp4

Found another issue of no grid snapping on select.js, next commit should fix it too.